gips/data/landsat/landsat.py

```diff
@@ -276,17 +279,23 @@ class landsatData(Data):
         'rad': {
             'assets': ['DN'],
             'description': 'Surface-leaving radiance',
-            'arguments': [__toastring]
+            'arguments': [__toastring],
+            'startdate': self._lt5_startdate,
```

Comment:
there's no `self` at this scope.  just a naked `_lt5_startdate` (or `_lc8_startdate` as the case may be).